### PR TITLE
doc(bindings): Update docs for Apple platforms.

### PR DESCRIPTION
**Pull request by pixlwave**
This is mainly small tweaks and including some info that was removed from the matrix-rust-components-swift package's README. I've also removed the old bash scripts now that we have the xtask for the SDK.

### CODE REVIEW

**Review by stefanceriu — APPROVED** [review]
Looks great 👍

File: bindings/apple/README.md [hunk 1]
```diff
@@ -6,24 +6,28 @@ This project and build script demonstrate how to create an XCFramework that can
 
 * the Rust toolchain
 * UniFFI - `cargo install uniffi_bindgen`
```
Comment (pixlwave) [inline bindings/apple/README.md:8]: Is this step still required, or can it be removed now?

Comment (stefanceriu) [reply bindings/apple/README.md:8]: From what I understand the xtask should use the bindgen package that is linked against the sdk, so no, I don't think this is required anymore

**Review by jplatte — APPROVED** [review]
Apart from the now extraneous UniFFI-bindgen install step, this looks good!

**Pull request merged**